src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -524,7 +524,7 @@ class SchedulerActions(
     * command, and constraints) are applied consistently across running
     * application instances.
     */
-  private def update(
+  private def update( // linter:ignore UnusedParameter
```

Comment:
Not in a method definition. you have to name them. I assumed this method existed for a reason, so i didn't remove the parameters (or the method)